versions.yaml

```diff
@@ -31,6 +31,7 @@ module-sets:
     modules:
       - go.opentelemetry.io/otel/log
       - go.opentelemetry.io/otel/sdk/log
+      - go.opentelemetry.io/otel/sdk/log/xlog
```

Comment:
Shouldn't this be excluded? We do not want to communicate stability of this module and it should definitely not be released in lockstep with the log sdk.

Reply:
These are experimental modules. We will move the other modules to stable modulset when stabilizing Logs and this module will remain here.

Reply:
I think experimental modules can follow other Go experimental modules and only be reference by commit hash. That communicates better the unstable nature clearer in my opinion.

Reply:
I like this idea. We can always decide later to release it :wink: 

https://github.com/open-telemetry/opentelemetry-go/pull/6271/commits/98d6811a38655f788d0d295b7c95d3cfa49c8bd1